src/lib/run-studies.ts

```diff
@@ -93,6 +93,16 @@ const _managementAppSampleData = {
 
 // Wrap calls in a function to avoid layers of promise resolves
 const main = async (): Promise<void> => {
+    let ignoreAWSRuns: boolean = false
```

Comment:
i think the `:boolean` is redundant when you initialize the value with a primitive type